### DISCUSSION TITLE
Where is the code?

### DISCUSSION THREAD

**Discussion by 1fabi0** [Digitaler-Impfnachweis/certification-apis#69] [Q&A] [answered]
All sources claim that this Project is OpenSource and Github was chosen but all we see here is some documentation about the api not the actual code of the api(of course without secrets). Or any other part (I think the Corona Warn App has it's part open source). So please either do not claim your project as open source or open source it. Right now it's lie if you say:" we have the source code puplished on Github for security reasons so everyone can check how secure everything really is and we can potentially close security breaches with the opens source comunity".

**Comment by oliver-steinbrecher** [comment, marked as answer]
We're going to publish the source code once we're live.
